quickstarts/gradle/daprhelloworld/zipkininstallation.txt

```diff
@@ -0,0 +1,14 @@
+-- create name space usernamespace
```

Comment:
Umm, why do we have this installation as a `.txt` file? Is it possible to convert it to a script?